lte/gateway/c/oai/tasks/nas/esm/DedicatedEpsBearerContextActivation.c

```diff
@@ -273,7 +279,27 @@ int esm_proc_dedicated_eps_bearer_context_accept(
           LOG_NAS_ESM, "ESM-PROC  - EBI %u was already ACTIVE\n", ebi);
       *esm_cause = ESM_CAUSE_PROTOCOL_ERROR;
     }
-    mme_app_handle_create_dedicated_bearer_rsp(ue_context_p, ebi);
+    bearer_context_t* bearer_ctx =
+        mme_app_get_bearer_context(ue_context_p, ebi);
+    /* Send MBR only after receiving ERAB_SETUP_RSP.
+     * bearer_ctx->enb_fteid_s1u.teid gets updated after receiving
+     * ERAB_SETUP_RSP.*/
+    if (bearer_ctx->enb_fteid_s1u.teid) {
+      mme_app_handle_create_dedicated_bearer_rsp(ue_context_p, ebi);
+    } else {
+      rc = esm_ebr_start_timer(
+          emm_context, ebi, NULL, ERAB_SETUP_RSP_TMR,
+          _erab_setup_rsp_tmr_exp_ded_bearer_handler);
+      if (rc != RETURNerror) {
+        OAILOG_DEBUG(
+            LOG_NAS_ESM,
+            "ESM-PROC  - Started ERAB_SETUP_RSP_TMR for "
+            "ue_id=" MME_UE_S1AP_ID_FMT
+            "ebi (%u)"
+            "\n",
```

Comment:
Nit: No need for extra new line, that is automatically added in all log messages.